.travis.yml

```diff
@@ -15,8 +11,10 @@ env:
 
 matrix:
   include:
-    - python: "3.6"
+    - python: "3.9"
       env: TOXENV=lint
+    - python: "3.9"
+      env: TOXENV=docs
```

Comment:
A `docs` task was defined in `tox.ini`, but wasn't defined in the travis config. This allows it to run regularly in CI.